system/reports/assets/simple.cfm

```diff
@@ -309,32 +311,52 @@ code {
 	<cfreturn bootstrapClass>
 </cffunction>
 
+<cffunction name="statusToEmoji" output="false">
+	<cfargument name="status">
+	<cfargument name="emojiService">
+
+	<cfset emoji="">
+
+	<cfif lcase( arguments.status ) eq "failed">
+		<cfset emoji=arguments.emojiService.get( 'warning' )>
```

Comment:
spacing please `emoji = `

Reply:
Fixed.

Reply:
Does this only apply to cfset? Otherwise it is inconsistent in the other where an equals sign is used.